local/local_test.go

```diff
@@ -133,11 +132,13 @@ func testLocalImage(t *testing.T, when spec.G, it spec.S) {
 			var repoName = newTestImageName()
 
 			it.Before(func() {
-				h.CreateImageOnLocal(t, dockerClient, repoName, fmt.Sprintf(`
-					FROM scratch
-					LABEL repo_name_for_randomisation=%s
-					LABEL mykey=myvalue other=data
-				`, repoName), nil)
+				baseImage, err := local.NewImage(repoName, dockerClient)
+				h.AssertNil(t, err)
+
+				h.AssertNil(t, baseImage.SetLabel("repo_name_for_randomisation", repoName))
```

Comment:
Is this label only needed for cache-busting on the daemon? Do we know if these are still needed?

Reply:
If we make this change we should be careful that we aren't relying on Force to paper over incomplete cleanup, to avoid leaking images.

Reply:
I did another pass but kept the `repo_name_for_randomisation` labels for now, as did I keep `busybox` as a base image since I don't quite understand the tests well enough to be sure what the requirements are for both. I might need a pair to get all the way there.